client/source/components/questionsAnswers/HelpfulA.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import * as requests from '../../utilities/axiosRequests.js';
+import { ButtonContainer } from './styles/Containers.styled';
+
+const HelpfulA = function CreateHelpfulAComponent({ id, getQuestions, helpfulness }) {
+  const [isReported, setIsReported] = useState(false);
+
+  const handleHelpfulA = function handleHelpfulA(event) {
+    event.target.setAttribute('disabled', true);
+    requests
+      .put(`/qa/answers/${id}/helpful`)
+      .then((results) => {
+        console.info(results.status);
+        getQuestions();
+      })
+      .catch((err) => console.error('handleHelpfulA error: ', err));
+  };
+
+  const handleReportedA = function handleReportedA(event) {
+    setIsReported(true);
+    event.target.setAttribute('disabled', true);
+    requests
+      .put(`/qa/answers/${id}/report`)
+      .then((results) => {
+        console.info(results.status);
+      })
+      .catch((err) => {
+        console.error('handleReportedA error: ', err);
```

Comment:
Thinking about this more, perhaps we should align how our handleReported functionality works? I noticed your logic here marks the link from "Report" to "Reported" upon clicking, and disables the link/button. My logic re-renders the review list, such that the reported review doesn't show up anymore. 

Curious to get your thoughts on which way you want us to align? I did some research on some other retail websites, and it looks more like your approach here.

Reply:
Oh yeah, I did notice that. I wasn't sure what yours did actually, but I thought this would be the best way for mine, and if there was an error, it could change back to "report" so all bases are covered. Let me know if you want to refactor yours or if I should try your way

Reply:
I like that rationale. I'll go ahead and align to yours, as it seems to match the BRD requirements, vs. the ReviewTile doesn't indicate as explicit requirements.